wgsl/index.bs

```diff
@@ -2943,26 +3111,22 @@ Issue: (dneto) Default rounding mode is an implementation choice.  Is that what
   <tr><td>`FOR`<td>for
   <tr><td>`FRAGMENT`<td>fragment
   <tr><td>`FUNCTION`<td>function
+  <tr><td>`HANDLE`<td>handle
```

Comment:
Is this needed?

Reply:
I moved it to reserved words.

If we use the full planned generality, then you can always get the base pointer of a variable and stuff it into a const.  But we could make a special rule for that. I added a TODO for that in the PR.